src/test/java/be/xplore/pricescraper/utils/ProductMatcherTests.java

```diff
@@ -0,0 +1,46 @@
+package be.xplore.pricescraper.utils;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@SpringBootTest
```

Comment:
It seems a bit excessive to make this a spring boot test. A `@BeforeEach` method would be sufficient to test this class.